### DISCUSSION TITLE
DeskHop v0.61

### DISCUSSION THREAD

**Discussion by hrvach** [hrvach/deskhop#112] [Announcements]
Bugfixes

- Fixed an issue with NKRO and consumer control
- Fixed stuck keys on reboot-and-disappear in Windows
- Improved device support 

NOTE: this is still a pre-release version, please report any bugs. Thank you!

<hr /><em>This discussion was created from the release <a href='https://github.com/hrvach/deskhop/releases/tag/v0.61'>DeskHop v0.61</a>.</em>

**Comment by chris-010** [comment]
This version fixed Caps lock stuck problem. #109

**Comment by OrpheeGT** [comment]
"Fixed stuck keys on reboot-and-disappear in Windows"

Is this the bug I have when booting Windows ?
On my laptop when it boots I have to disconnect deskhop from my USBC/HDMI hub until Windows session is logged on or my hub seems totally disconnected.

Thanks :)

**Comment by Bunny-Head** [comment]
Amazing release! May I ask what's the key to swap between screens now? And will te be able to configure using WebUI in the future?

BTW, I just tried some FPS games and same issues is happening (I know this hasn't been addressed yet, just testing if by any chance and by happy accident it got fixed :D).

Again, infinite thanks for your amazing work!

**Reply by OrpheeGT** [reply]
Current RC build has F24 set to switch keyboard shortcut (means none)

https://github.com/hrvach/deskhop/blob/rc/src/include/user_config.h#L35

And I don't see any shortcut for it on GUI.

Unless you build RC yourself and set value back to main release ```HID_KEY_CAPS_LOCK``` you won't have it currently.

**Comment by OrpheeGT** [comment]
@hrvach OMG !!!

I forked your repo, and built myself RC branch !

It built only one common deskhop.uf2 instead of boadA and boardB.

Now I have my keyboard from https://github.com/hrvach/deskhop/issues/45 working fine,
and ```LShift + RShift + c + o``` works to and load board A as USB key with config.htm available !

(Any way to have ```LShift + RShift + c + o``` work on active screen instead of board A ?)

I was just too dumb and took files from your source ZIP instead of actual  direct deskhop.uf2...

**Comment by OrpheeGT** [comment]
@hrvach It seems I have an issue with config menu.

When I click on "Read" and choose the deskhop.
It works but menu is partially filled : 

![image](https://github.com/hrvach/deskhop/assets/8038551/21226303-0b1d-40ae-9427-789256011151)

I have to "wipe settings" and "read" again to see all correctly filled : 
![image](https://github.com/hrvach/deskhop/assets/8038551/83b67eee-f986-449e-9a0d-f6d92db6b1ec)

Once I exit from config/usb mode.
Bug appears again when I go back to check/modify settings.

And, as you can see on the screen, sometime it detect my Windows board A as a Mac OS...

And, on Ubuntu I can't connect to deskhop when I try to "read" (using Chromium Version 126.0.6478.114 (Build officiel) snap (64 bits)). The popup show the deskhop, but can't connect to it it just closes the popup, but nothing read.
Whereas it works on Windows laptop.

Edit : Actually the bug appens if I click on read instead of "Connect" ! maybe a user error.

**Reply by hrvach** [reply]
I have an idea what might be happening, I'll have to rework the reading section to avoid any chance of this class of bugs in the future. Thanks for noticing!

**Reply by OrpheeGT** [reply]
I understand there is no more key shortcut to enable/disable "screensaver", setting is now only available from GUI.
I like to be able to enable/disable it on the fly... even if I really like to have it on GUI for some friend not confident with shorcuts :)

**Reply by Bunny-Head** [reply]
Would be nice to have a tab on the GUI to activate/deactivate the shortcuts and to map/remap these (I really need to change the lock screen key XD).

**Comment by OrpheeGT** [comment]
Hi,
May I also ask to set anti idle in seconds or minutes instead of micro seconds ?

I tried to set max time to 6 hours, and after save, it seems the max time set was like 25 minutes.
